monitoring/uss_qualifier/scd/test_definitions/CHE/automated_test/automated_test_1.json

```diff
@@ -0,0 +1,659 @@
+                                    "format": "RFC3339"
+                                }
+                            }
+                        ],
+                        "state": "Accepted",
+                        "off_nominal_volumes": [],
+                        "priority": 0
+                    }
+                },
+                "known_responses": {
+                    "acceptable_results": [
+                        "ConflictWithFlight"
+                    ],
+                    "incorrect_result_details": {
+                        "Rejected": {
+                            "test_code": "nominal_planning_test",
+                            "relevant_requirements": [],
+                            "severity": "High",
+                            "subject": "",
+                            "summary": "Injection request for a valid flight was unsuccessful",
+                            "details": "All operational intent data provided was complete and correct with no airspace conflicts. The operational intent data should have been processed successfully and flight should have been planned."
+                        },
+                        "Failed": {
+                            "test_code": "nominal_planning_test",
+                            "relevant_requirements": [],
+                            "severity": "High",
+                            "subject": "",
+                            "summary": "Injection request for a valid flight was unsuccessful",
+                            "details": "All operational intent data provided was complete and correct with no airspace conflicts. The operational intent data should have been processed successfully and flight should have been planned."
+                        },
+                        "Planned": {
+                            "test_code": "nominal_planning_test",
+                            "relevant_requirements": [
+                                "A operational intent that has time or space conflict should not be planned by the USS"
+                            ],
+                            "severity": "High",
+                            "subject": "Operational Intent provided should not be sucessfully planned by the USSP",
+                            "summary": "The operational intent details provided were generated in such a way that they should not have been planned.",
+                            "details": "The co-ordinates of the 4D Operational intent conflicts with an existing operational intent in the area and the processing result should not be a successful planning of the intent."
+                        }
+                    }
+                },
+                "injection_target": {
+                    "uss_role": "Second USS"
+                },
+                "name": "za0q1nwk"
+            }
+        },
+        {
+            "name": "Delete first injected flight",
```

Comment:
I'd recommend that we don't need this step -- uss_qualifier should attempt clean up after every test, so if there is still an injected flight that hasn't been deleted, it should automatically detect that and try to fix it (no need to explicitly say to delete it since the implicit process will handle it).  Same with other automated test.